### ISSUE TITLE
For proposals in rounds with two applications, the ability/need to dismiss regardless of current state

### ISSUE THREAD

**Issue by danblah** [OpenTechFund/opentech.fund#638]
https://apply.opentech.fund/apply/submissions/314/ as an example where this submission needs to be dismissed and the applicant won't be submitting a draft for team review.

**Comment by frjo** [comment]
@danblah  So it happens that you invite for proposal but somethings comes up and you need dismiss it even if the applicant have not submitted it yet?

**Comment by danblah** [comment]
That's right. Each round there are a number of submissions for a number of reasons that come up (we've already a couple this current round that are this way).
